test/unit/inbound/process-return-settlement.test.js

```diff
@@ -22,15 +22,22 @@ const getSettlementByInvoiceNumberAndValue = require('../../../app/inbound/retur
 jest.mock('../../../app/inbound/return/save-settlement')
 const saveSettlement = require('../../../app/inbound/return/save-settlement')
 
+jest.mock('../../../app/inbound/return/save-schedule')
+const saveSchedule = require('../../../app/inbound/return/save-schedule')
+
 const processReturnSettlement = require('../../../app/inbound/return')
 
 let settlement
+let savedSettlement
 
 describe('process return settlement request', () => {
   beforeEach(() => {
     settlement = JSON.parse(JSON.stringify(require('../../mock-settlement')))
+    savedSettlement = { settlementId: 1 }
```

Comment:
I think we can have this as `{...settlement, settlementId: 1 }`

I assume this is what that will return when given `settlement`?